book.rb

```diff
@@ -11,6 +11,7 @@ def initialize(title, author)
 
   def add_rental(person, date)
     Rental.new(date, self, person)
+    @rental.push(Rental.new(date, self, person))
   end
 
```

Comment:
- [ **OPTIONAL** ] `README.md` file: I think it would be better if you update the the `Getting Started` section by adding some information on how to `clone` your repo, `run tests`, `install gems(if exist)`,...
To help you you use this **[article](https://gist.github.com/tresorsawasawa/4cde69cff1b40161b9de380f9891f415)** as example